.travis.yml

```diff
@@ -33,3 +33,7 @@ matrix:
     rust: nightly
     before_script: cd rust
     script: cargo build --no-default-features
+  - language: csharp
```

Comment:
Perhaps you could set up a Windows build using AppVeyor?

https://github.com/miscreant/miscreant/blob/master/appveyor.yml

Reply:
Sure, I'll do that, too.